### ISSUE TITLE
manage-post-tags mangles front matter

### ISSUE THREAD

**Issue by RNCTX** [zhuochun/md-writer#212]
Atom v1.19.0 on OSX, md-writer v2.6.5

I have the following config regarding front matter as a template...

 ```
 "markdown-writer":
    fileExtension: ".md"
    frontMatter: '''
      ---
      date: <date> CDT
      title: <title>
      category: post
      tags:
      facebook:
      twitter:
      ---
    '''
```

My siteEngine is set to jekyll, I'm using v3.1.6

I can create a new post draft, using the new-draft function and it shows up normally. I also have tags and categories linked via json from a live website per the guide in your docs, and the sync works normally, here's the result of a new draft...

<img width="290" alt="screen shot 2017-08-16 at 4 38 59 pm" src="https://user-images.githubusercontent.com/13775820/29386576-2f67043e-82a2-11e7-8d2b-7c7f0cddab11.png">

If I add a tag via manage-post-tags, manage-post-tags single quotes the date and adds `null` to the fields I previously left blank.  

<img width="321" alt="screen shot 2017-08-16 at 4 39 16 pm" src="https://user-images.githubusercontent.com/13775820/29386649-731964c4-82a2-11e7-8e16-7d68e491cfc4.png">

The nulls are a minor inconvenience, I can have liquid look for != 0 and != 'null' to properly exclude those from parsing where they are supposed to be excluded, but single quoting the date field makes jekyll miss the post entirely.

**Comment by zhuochun** [comment]
Afaik, Jekyll can parse the YAML written out by markdown-writer plugin (but I agree write out null is stylish).

Quoted string is valid YAML. I think the date was missed due to the date format is not correct. Try remove the `CDT` timezone.

**Comment by RNCTX** [comment]
Thanks for the reply.

I just tried removing the timezone, but the single quote is still there.  However, removing the time zone did cause jekyll to pick the post up rather than ignoring it.  I suppose I found an intricacy of jekyll ;).  It will pick up a timezone not quoted, or pick up a proper string without a time zone quoted, but not both an incorrect _and_ quoted string.

I'm considering this closed and not a bug.
